README.md

```diff
@@ -110,6 +110,15 @@ factories along with this extension's factories using this statement:
 SolidusDevSupport::TestingSupport::Factories.load_for(SolidusBolt::Engine)
 ```
 
+#### Special Tests
+
+A few tests in the test suite require some additional steps to execute successfully when they are modified. These are listed below along with the steps needed to execute these tests.
+
+- `/spec/services/solidus_bolt/accounts/detail_service_spec.rb`
+This test requires a valid bolt access token to execute successfully. Generate the bolt access token by logging into a bolt sandbox account in development and copying the token from the `session['bolt_access_token']` key.
```

Comment:
I'm not clear I understand it myself by reading this. Maybe some more details or a link would help with clarity

Reply:
I don't really have a link for this, but could you let me know what extra details would you like in this explanation to make it more easy yo understand.

Reply:
Modified here
[6ad4d41](https://github.com/nebulab/solidus_bolt/commit/6ad4d41739310671594d966c6a2b5c25c3edbade)